### DISCUSSION TITLE
WOULD THE RARE INSTANCES OF MISLABELED IMAGES AFFECT THE MODEL?

### DISCUSSION THREAD

**Discussion by MuhammadBilal848** [ultralytics/ultralytics#14590] [Q&A]
I have like 5k images and I have 5 classes and I annotate all of them and some photos are incorrectly labeled(say 50 or 60 or less). Would it affect the model's performance?

**Comment by pderrenger** [comment]
Yes, mislabeled images can affect the model's performance, especially if the errors are significant or if the mislabeled instances are similar to correctly labeled ones. However, with a dataset of 5,000 images, having 50-60 mislabeled images is relatively minor and may not drastically impact overall performance. It's always best to correct as many errors as possible to ensure the highest model accuracy. If you notice performance issues, consider reviewing and correcting the annotations.
